### PR TITLE
Some spelling and grammatical mistakes

### PR DESCRIPTION
**Pull request by kunakl07**
I have fixed some spelling and grammatical mistakes

### CODE REVIEW

File: doc/theory/spherical.rst [hunk 1]
```diff
@@ -26,31 +26,30 @@ OP
 radial distance
    or radius.  The Euclidean length of OP.
 
-z axis
-   The vertical of the sphere.  If we consider the sphere as a globe,
-   then the z axis runs from south to north.  This is the zenith direction of the sphere. 
+z-axis
+   The vertical of the sphere.  If we consider the sphere like a globe,
```
Comment (arokem) [inline doc/theory/spherical.rst]: I actually prefer this one as it already is.

```suggestion
   The vertical of the sphere.  If we consider the sphere as a globe,
```

File: doc/theory/spherical.rst [hunk 2]
```diff
@@ -103,7 +102,7 @@ Matlab has functions ``sph2cart`` and ``cart2sph``.  These use the terms
 physics and mathematics conventions.   Here ``theta`` is the azimuth angle, as
 for the mathematics convention, but ``phi`` is the angle between the reference
 plane and OP.  This implies different formulae for the conversions between
-Cartesian and spherical coordinates that are easy to derive.
+Cartesian and spherical coordinates are easy to derive.
```
Comment (arokem) [inline doc/theory/spherical.rst]:
```suggestion
Cartesian and spherical coordinates that are easy to derive.
```

Here too. I would leave as-is

File: doc/theory/spherical.rst (hunk 1 above)
Comment (kunakl07) [reply doc/theory/spherical.rst]: Oh sure. I will do the changes.

File: doc/theory/spherical.rst (hunk 2 above)
Comment (kunakl07) [reply doc/theory/spherical.rst]: Oh cool. Will revert it back
Thank you

**Comment by kunakl07** [comment]
> Thanks! I would ask to revert a couple of these changes, but otherwise, GTG. GGT!

Yeah sure.Would do them

**Comment by skoudoro** [comment]
Same question here, is it good to go for you @arokem? Feel free to merge this PR. Thanks!

**Comment by arokem** [comment]
Sure. Onwards and upwards!

**Pull request merged**